protocol.go

```diff
@@ -22,6 +22,13 @@ const (
 	Error   ResponseType = 2
 )
 
+type ProtocolType = rune
+
+const (
+	Chmod ProtocolType = 'C'
```

Comment:
Not sure if it is called a `Chmod` message, might be a confusing term since the message is also used for signaling the size of the file and also the start of its transfer. I would keep it if it is part of some standard terminology, otherwise maybe call it `Create`?

Reply:
Or `Copy`.